doc/source/rllib-training.rst

```diff
@@ -86,6 +86,22 @@ You can control the degree of parallelism used by setting the ``num_workers`` hy
 .. Original image: https://docs.google.com/drawings/d/14QINFvx3grVyJyjAnjggOCEVN-Iq6pYVJ3jA2S6j8z0/edit?usp=sharing
 .. image:: rllib-config.svg
 
+Scaling Guide
+~~~~~~~~~~~~~
+
+Here are some rules of thumb for scaling training with RLlib.
+
+1. If the environment is slow and cannot be replicated (e.g., since it requires interaction with physical systems), then you should use a sample-efficient off-policy algorithm such as `DQN <rllib-algorithms.html#deep-q-networks-dqn-rainbow-parametric-dqn>`__ or `SAC <rllib-algorithms.html#soft-actor-critic-sac>`__. These algorithms default to ``num_workers: 0`` for single-process operation. Consider also batch RL training with the `offline data <rllib-offline.html>`__ API.
+
+
+2. If the environment is fast and the model is small (most models for RL are), use time-efficient algorithms such as `PPO <rllib-algorithms.html#proximal-policy-optimization-ppo>`__, `IMPALA <rllib-algorithms.html#importance-weighted-actor-learner-architecture-impala>`__, or `APEX <rllib-algorithms.html#distributed-prioritized-experience-replay-ape-x>`__. These can be scaled by increasing ``num_workers`` to add rollout workers. It may also make sense to enable `vectorization <rllib-env.html#vectorized>`__ for inference. If the learner becomes a bottleneck, multiple GPUs can be used for learning by setting ``num_gpus > 1``.
```

Comment:
recommend you use https://www.sphinx-doc.org/en/1.5/markup/inline.html#cross-referencing-arbitrary-locations (makes this much more maintainable).